kumascript/macros/NonStandardBadge.ejs

```diff
@@ -13,6 +13,6 @@ var titleAttrValue = mdn.localString({
     "en-US": "This API has not been standardized."
 });
 %>
-<svg class="icon icon-nonstandard" tabindex="0">
-    <use xlink:href="/assets/badges.svg#icon-nonstandard"></use>
-</svg>
+<abbr style="mask-size: 14px;" class="only-icon icon icon-nonstandard" title="Non-Standard">
```

Comment:
I believe this should be equivalent to `mask-size: 1rem`, given that the sidebar has a font-size of 14px.

If so, it would probably make sense to add this as `mask-size: var(--size);` to `.icon` in `client/src/ui/atoms/icon/index.scss`.

Reply:
https://github.com/mdn/yari/blob/main/client/src/app.scss#L24 

1 rem is 16px, the root font size.

Reply:
1em works

Reply:
You could try setting `--size` to `1em` instead of `1rem` in `client/src/ui/atoms/icon/index.scss`.

Intuitively `1em` makes more sense than `1rem`.

Reply:
That does nothing if I keep --icon-size in the --size calculation. If I remove it, it works perfectly for the sidebar, but the question is if what it does to the compatibility table is okay.

Before:
![before](https://user-images.githubusercontent.com/29206584/158702910-cdee7371-9faa-4529-9800-0f591eacef34.png)

After:
![after](https://user-images.githubusercontent.com/29206584/158702916-3d386ee6-2a89-4f56-a805-8606b94d3c24.png)

Reply:
The reason for the change is of course the sides of the 16x14 triangle being cut off in the 14x14 element, so the other option is of course to maked the triangle 14x12.25

Reply:
Changed .sidebar.icon